shop/.dockerignore

```diff
@@ -9,3 +9,6 @@
 !composer.*
 !yarn.*
 !.stylelint*
+!tests/
+!setup_test
```

Comment:
I would prefere something like `!*test*`, so we could cover more naming schemas in shop implementations